### ISSUE TITLE
Docs: Starter template location incorrect on website

### ISSUE THREAD

**Issue by whoabuddy** [forestryio/forestry.io#218]
## TL;DR

I was trying to set up the Parsa template via the Forestry website, but the location of the GitHub repository was incorrect. The error was not immediately visible (from a new user perspective), but after looking at the website vs the Forestry GitHub page, I found that the path to the repository is incorrect.

## To reproduce:

1. Navigate to Forestry.io https://forestry.io/
2. Click on Select a Starter Template https://forestry.io/starters/
3. Select Parsa on Hugo, which opens up the Create a Site dialog (pictured below)

![2020-07-30 07 07 58 app forestry io a4d1fba53af0](https://user-images.githubusercontent.com/9038904/88932837-71a49d00-d233-11ea-8fb1-18e3c083f837.png)

From there, it was not immediately obvious that the template had an error in its path, and I continued to follow the steps at the bottom to connect my GitHub account. My assumption was that the red X at the top came from the fact I was only on Step 1/2 at the bottom.

![2020-07-30 07 08 36 app forestry io 0ba995e8ea8a](https://user-images.githubusercontent.com/9038904/88932939-9dc01e00-d233-11ea-9530-2bf1c1a008d9.png)

![2020-07-30 07 09 25 app forestry io 3e1fbc035364](https://user-images.githubusercontent.com/9038904/88932949-9f89e180-d233-11ea-89de-fbd287ffb272.png)

After connecting my GitHub account, I found that I was able to select my repository name, but I was unable to continue using the Import button at the bottom. I felt stuck and switched from my mobile browser to desktop to do further troubleshooting, only to have the same result.

![2020-07-30 07 10 40 app forestry io 732650bcf794](https://user-images.githubusercontent.com/9038904/88933080-c9db9f00-d233-11ea-814c-c705209b92ec.png)

Finally, after hovering over the red X by the template, I could see the error `Could not connect to repository`. Looking at this from the Forestry GitHub page, I was able to locate the Parsa template, but the name of the repository has a slightly different format:

- on Website: git@github.com:forestryio/parsa-hugo-forestry.git
- on GitHub: git@github.com:forestryio/hugo-parsa-forestry.git

This may affect other templates as well. Using the `Import to Forestry` button in the GitHub readme, I was able to set up the repository and template as expected with the same flow.

![2020-07-30 07 13 55 github com 1b4e2371d6f5](https://user-images.githubusercontent.com/9038904/88933460-4a9a9b00-d234-11ea-9bcb-78395d35be73.png)

**Comment by DirtyF** [comment]
Thanks for feedback, should be fixed now 😊
